multiparty-sendrecv/gst/mp-webrtc-sendrecv.c

```diff
@@ -180,7 +181,9 @@ on_incoming_stream (GstElement * webrtc, GstPad * pad, GstElement * pipe)
       G_CALLBACK (on_incoming_decodebin_stream), pipe);
   gst_bin_add (GST_BIN (pipe), decodebin);
   gst_element_sync_state_with_parent (decodebin);
-  gst_element_link (webrtc, decodebin);
+
+  sinkpad = gst_element_get_static_pad (decodebin, "sink");
+  gst_pad_link (pad, sinkpad);
```

Comment:
You also have to `gst_object_unref(sinkpad)` once you're done here, otherwise the pad is leaked.

Otherwise this is the right thing to do :)

Reply:
Fixed the unref, thanks for pointing out.

Reply:
Thanks but you only fixed it in one place :)

Reply:
now also in the second place ;-)

Reply:
Thanks :)